projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -1055,6 +1055,14 @@ dataJob:
         # Options are "DB" for database and "K8S" for kubernetes. The control service will read
         # properties from the specified source.
         readDataSource: "K8S"
+      synchronizationTask:
```

Comment:
Shouldn't deployment yaml also be edited? How are those passed to the application ?

Reply:
Correct, I will update it